### PR TITLE
Move base image to new heredoc format

### PR DESCRIPTION
**Pull request by lfrancke**
# Description

Move base image to new heredoc format
There are no functional changes

### CODE REVIEW

File: stackable-base/Dockerfile [hunk 1]
```diff
@@ -49,63 +53,86 @@ SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
...
-    echo "alias ..='cd ..'" >> /stackable/.bashrc && \
-    echo "export PS1='\u@\[\e[36m\]\H\[\e[m\] \[\e[32m\]\$(pwd)\[\e[m\] \\$ '" >> /stackable/.bashrc && \
-    echo -e "if [ -f ~/.bashrc ]; then\n\tsource ~/.bashrc\nfi" >> /stackable/.profile && \
-    chown stackable:stackable /stackable/.bashrc && \
-    chown stackable:stackable /stackable/.profile
+
+# echo won't expand escape sequences. Consider printf.
+# https://github.com/koalaman/shellcheck/wiki/SC2028
+# Reason: This is complaining about the complicated PS1 statement.
+# While it seems like black it seems to work as intended so I'm not going to touch it!
```
Comment (razvan) [inline stackable-base/Dockerfile:60]:
```suggestion
# It seems to work as intended so I'm not going to touch it!
```

File: stackable-base/Dockerfile [hunk 2]
```diff
@@ -49,63 +53,86 @@ SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
...
+  nano \
+  tar
+
+# Added only temporarily to create the user and group, removed again below
+microdnf install shadow-utils
+groupadd --gid 1000 --system stackable
+useradd --gid stackable --uid 1000 --system stackable -d /stackable
+
+mkdir /stackable
+chown stackable:stackable /stackable
```
Comment (razvan) [inline stackable-base/Dockerfile:99]: I always wondered why we don't properly assign `/stackable` as home dir to the `useradd` command and let it do the right thing.

Comment (lfrancke) [reply stackable-base/Dockerfile:99]: I can try to look at that in a follow-up PR. I wanted this to be purely "formatting" changes.

**Review by razvan — APPROVED** [review]

File: stackable-base/Dockerfile (hunk 2 above)
Comment (lfrancke) [reply stackable-base/Dockerfile:99]: It looks like we already do and these two lines might actually be redundant. I'll check!

**Pull request merged**